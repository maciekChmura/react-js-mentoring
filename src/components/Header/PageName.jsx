import React from 'react';
import styled from 'styled-components';

const H1 = styled.h1`
  grid-area: 2 / 3 / 1 / 2;
  font-size: 28px;
  color: #dd1133;
  width: auto;
  /* padding: 10px; */
`;

const PageName = () => <H1>Netflix Roulette</H1>;

export default PageName;
