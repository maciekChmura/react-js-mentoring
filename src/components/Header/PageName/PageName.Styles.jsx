import styled from 'styled-components';

const H1 = styled.h1`
  grid-area: 2 / 3 / 1 / 2;
  font-size: 28px;
  color: #dd1133;
  width: auto;
`;

export default H1;
