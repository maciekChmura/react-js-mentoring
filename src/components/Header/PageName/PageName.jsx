import React from 'react';
import H1 from './PageName.Styles';

const PageName = () => <H1>Netflix Roulette</H1>;

export default PageName;
