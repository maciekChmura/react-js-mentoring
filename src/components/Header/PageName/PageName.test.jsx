import React from 'react';
import renderer from 'react-test-renderer';
import PageName from './PageName';

test('PageName renders', () => {
  const component = renderer.create(<PageName />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
