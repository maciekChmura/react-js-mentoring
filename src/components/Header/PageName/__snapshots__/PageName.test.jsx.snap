// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageName renders 1`] = `
<h1
  className="PageNameStyles__H1-sc-1qqo73k-0 fMwOSL"
>
  Netflix Roulette
</h1>
`;
